plugin/src/main/java/com/fourlastor/pickle/PicklePlugin.kt

```diff
@@ -83,30 +84,42 @@ class PicklePlugin : Plugin<Project> {
         var packageName: String? = null
         var featuresDir: String? = null
         var strictMode: Boolean = true
+        var androidTest: Boolean = true
+        var unitTest: Boolean = false
     }
 
     override fun apply(project: Project) {
         val extension = project.extensions.create("pickle", Extension::class.java)
 
         project.afterEvaluate {
-            project.plugins.all {
-                when (it) {
+            project.plugins.all { plugin ->
+                when (plugin) {
                     is LibraryPlugin -> {
                         project.extensions.findByType(LibraryExtension::class.java)?.run {
```

Comment:
I wonder if you can do 

```
project.extensions.findByType(TestedExtension::class.java)?.run {
```

here?

Reply:
this will enable us to merge 2 branches. And also probably will support other plugins like `instant-apps`, `dynamic-delivery` etc.

Reply:
Good point but it requires a bit more testing, I'll open an issue

Reply:
Definitely makes sense